api/STYLE.md

```diff
@@ -132,13 +134,15 @@ To add an extension config to the API, the steps below should be followed:
        )
    ```
 1. Add to the v2 extension config proto `import "udpa/annotations/migrate.proto";`
-2. Add to the v2 extension config proto a package level `option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.filters.http.foobar.v3";`.
-   This places the filter in the correct [v3 package hierarchy](#package-organization).
-3. Add a reference to the v2 extension config in (1) in [api/docs/BUILD](docs/BUILD).
-4. Run `./tools/proto_format fix`. This should regenerate the `BUILD` file,
+1. Add to the v2 extension config proto a file level `option (udpa.annotations.file_migrate).move_to_package = "envoy.extensions.filters.http.foobar.v3";`.
```

Comment:
Per your comment about about vNalpha, I'm confused. Are we back to saying that it's OK to use alpha for a proto? If so do we need this annotation? Or do we prefer to do it also? It seems redundant? I thought we had this annotation because we couldn't do the alpha. Either way can you potentially clarify the docs and have a worked alpha example below if that is what we are doing?

Reply:
So this addresses my own comment [here](https://github.com/envoyproxy/envoy/pull/9826#pullrequestreview-353544660). The annotations seems redundant now, but it has doc effect, also it will help when we stabilize v4alpha to v4. When we boost v4alpha, we don't have information whether a proto is boosted from `v3alpha` or `v3`, when we stabilize `v4alpha` to `v4` we can exclude protos with this annotation.

This also gives a chance to stabilize proto during major upgrade: i.e. removing annotation from an v3alpha proto file will make it to v4 during the major upgrade.

Reply:
OK that's fine with me if you think it works. Can you update the docs a bit to have an example of both cases? I think that would be helpful.

Reply:
@mattklein123 done and add link to example filters.